census/templates/index.html

```diff
@@ -32,6 +32,10 @@
                             <option value="{{ language.name }}" {% if language.selected %} selected="selected" {% endif %}>{{ language.name }}</option>
                         {% endfor %}
                     </select>
+                    <input id="form-day" type="hidden" name="day" value="{{ request.day }}" />
```

Comment:
Hidden fields which get populated when the user selects a date on the datepicker.